molecularnodes/nodes/material.py

```diff
@@ -187,3 +189,249 @@ def __init__(self, **kwargs):
 class TransparentOutline(MaterialConstructor):
     def __init__(self, **kwargs):
         super().__init__("MN Transparent Outline", **kwargs)
+
+
+class MN_Materials:
```

Comment:
This shouldn't need `MN_` prefix as it is coming from the MN package.

Reply:
Updated .